config/routes.yaml

```diff
@@ -33,6 +33,14 @@ list-comments.html:
     methods: [GET]
     controller: 'WMDE\Fundraising\Frontend\App\Controllers\Donation\ListCommentsController::handleHtml'
 
+Unterstützerliste-redirect:
+    path: '/page/Unterstützerliste'
+    methods: [GET]
+    controller: 'Symfony\Bundle\FrameworkBundle\Controller\RedirectController'
+    defaults:
+        path: '/page/hall-of-fame'
+        permanent: true
+
```

Comment:
Please add a comment that this is temporary and needs to be removed when manitu has adapted the web sever config to do the redirect

We don't want the framework to do stuff a web server/load balancer can do more efficiently.